scripts/prepare_automatic_backups.py

```diff
@@ -34,6 +34,16 @@
 
 
 def generate_backup_url(cloud_storage_bucket_name, module_class_names):
+    """Generates backup url of new added urls in cron file.
+
+    Args:
+        cloud_storage_bucket_name: str. Name of cloud storage bucket.
+        module_class_names: list(str). List of newly added module and class
+        names.
+
+    Returns:
+        str. Returns backup url.
```

Comment:
Be consistent please -- for Returns docstrings just describe the object that is being returned. You do not need to start the description with "Returns".